src/networks.ts

```diff
@@ -20,9 +20,13 @@ export interface UtxoNetwork extends BaseNetwork {
   pubKeyHash: number;
   scriptHash: number;
   wif: number;
+  explorerUrl?: string;
+}
+
+export interface AccountNetwork extends BaseNetwork {
+  explorerUrl?: string;
```

Comment:
Since this is optional anyway, we should just add it to BaseNetwork and set `explorerUrl = undefined` for coins which don't have an explorer.  Advantage with that approach is we get a compile error if we forget to define the explorerUrl for a network.